src/styles/globals.css

```diff
@@ -42,7 +42,7 @@
   }
 
   .sidebar-close-button {
-    top: env(safe-area-inset-top);
+    top: env(safe-area-inset-top) !important;
```

Comment:
If you are removing the `top-0` class from the classes is `!important` still required here?

Reply:
Yes because it's !important

Reply:
I'll remove it